src/sort-filter-bar/sort-filter-bar.ts

```diff
@@ -130,6 +130,9 @@ export class SortFilterBar
     return html`
       <div id="container">
         <section id="sort-bar" aria-label="Sorting options">
+          <div>
+            <slot name="sort-slot-left"></slot>
+          </div>
```

Comment:
Q: Curious whether we can do away with the wrapper div here?

Reply:
removed div wrapper